packages/site/src/layout/SearchBox.tsx

```diff
@@ -41,7 +41,7 @@ export const SearchBox = ({
   const filteredComponentList = useMemo(() => {
     return componentList.filter(
       d =>
-        d.title.includes(search) ||
+        d.title.toLowerCase().includes(search.toLowerCase()) ||
         d.additionalMatches?.find(e => e.includes(search)),
```

Comment:
@scotttjob I feel like I just need to add the `toLowerCase` somewhere on L45 but I tried plugging it in a few places and couldn't get the results I was expecting:

`d.additionalMatches?.find(e => e.includes(search.toLowerCase))`

![image](https://github.com/user-attachments/assets/3b4dfe44-5d24-4a66-a332-7a80851c62e6)
![image](https://github.com/user-attachments/assets/dc40e18b-33a9-4920-87d1-ef7d5c538ea5)

`d.additionalMatches.toLowerCase()?.find(e => e.includes(search.toLowerCase))`

![image](https://github.com/user-attachments/assets/9ce79d3d-313b-4896-8abb-b84b7390b3dc)